vroom/buffer.vroom

```diff
@@ -9,6 +9,23 @@ But before we dive into things, let's get maktaba installed:
   :let g:bootstrapfile = g:maktabadir . '/bootstrap.vim'
   :execute 'source' g:bootstrapfile
 
+Vim is missing a way to look up the text under a visual selection, for use in
+visual mode mappings for instance. maktaba#buffer#GetVisualSelection provides
+that.
+
+  % Line 1<CR>
+  |Line 2<CR>
+  |Line 3<CR>
+  |Line 4<CR>
+  |Line 5<CR>
+  |Line 6
+
+  > 3gg5|v4gg$
+  :<C-U>echomsg string(split(maktaba#buffer#GetVisualSelection(), "\n", 1))
+  ~ [' 3', 'Line 4']
+
```

Comment:
Could you add a test using visual line mode (`V`)? That reports 2^31-1 for the `'>` column number, which is weird enough that it seems worth covering (though it looks like it'll work fine).